.github/ISSUE_TEMPLATE/backend_story.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Backend Story
+about: Explain the User story, regarding Backend API related changes.
+title: 'Backend Story'
+labels: backend-story, user-story
+assignees: ''
+
+---
+
+<!-- /!\ Please ensure that you are NOT disclosing any customer information without their consent /!\ -->
+
+## User Story
+As an Automate User, I will use this api to save this data.
+
+## Acceptance Criteria
+- This should have this api working with these inputs.
+- Should produce this output
+
+## Implementation Details
+- In this file, add this function.
+
+## Definition of Done
+- All things specified in User Story Acceptance Criteria should be fulfilled. 
+- All Exceptions are Handled Properly 
+- Ensure logs have no unnecessary data. 
+- Test coverage for the new feature is done to at least 70% 
+- Docs changes PR is Raised. 
+- Swagger Documentation updated 
+- Smoke Test done. 
+- Ensure Build and Integration Pipelines are Green. 
+- PR has 2 approvers. 
+- All Code Review Comments are Resolved. 
```

Comment:
Component README doc needs to be updated.

Reply:
done